### PR TITLE
Enterprise SSO

### PR DESCRIPTION
**Pull request by CamrynCarter**
First draft of Enterprise SSO documentation. Screenshots are temporary and will need to be replaced.

### CODE REVIEW

File: content/docs/overview/identity-and-access-management/index.md [hunk 1]
```diff
@@ -70,4 +70,28 @@ To set your `Tenant Display Name`:
...
+[Enterprise customers](https://www.rkvst.com/pricing/) may use their preferred Identity Provider (IDP) to sign-on to RKVST.
+
+1. Navigate to `Manage RKVST` on the sidebar and select `Tenancy`.
+
+Enter your SSO configuration, then select `SAVE ENTERPRISE SSO CONFIG`. 
+
+{{< img src="SSOConfig.png" alt="Rectangle" caption="<em>Configure SSO</em>" class="border-0" >}}
+
+{{< note >}}
+**NOTE:** To retrieve the necessary data for the configuration form, your IDP must be configured to be compatible with RKVST.
```
Comment (kjc-jitsuin) [inline content/docs/overview/identity-and-access-management/index.md:86]: This seems a little squahed in, can we put better spacing below it?

**Review by kjc-jitsuin — COMMENTED** [review]
Are we getting a better screenshot for these? I'm not sure we want to advertise Lastpass, or let everyone spam Waldek's mail!

Otherwise all good.

**Comment by CamrynCarter** [comment]
> Are we getting a better screenshot for these? I'm not sure we want to advertise Lastpass, or let everyone spam Waldek's mail!
> 
> Otherwise all good.

Yes, all the screenshots will be replaced (these were placeholders). Waiting on engineering to send, I will remind early next week.

File: content/docs/overview/identity-and-access-management/index.md [hunk 2]
```diff
@@ -70,4 +70,41 @@ To set your `Tenant Display Name`:
 
 #### Verified Domain 
 
-The second way to set your tenancy name is to [get your domain verified](../../beyond-the-basics/verified-domain) by the RKVST team. Your verified domain name will be visible to the people you share information with and will be publicly available if you create a Public Asset.
\ No newline at end of file
+The second way to set your tenancy name is to [get your domain verified](../../beyond-the-basics/verified-domain) by the RKVST team. Your verified domain name will be visible to the people you share information with and will be publicly available if you create a Public Asset.
+
+## Enterprise Single Sign-On
+
+[Enterprise customers](https://www.rkvst.com/pricing/) may use their preferred Identity Provider (IDP) to sign-on to RKVST.
```
Comment (honourfish) [inline content/docs/overview/identity-and-access-management/index.md:77]: I think its important to state before you can configure an esso, the tenancy needs to have a verified domain?

Also they need to have a cap of >0.

Without both of those the configuration options won't show in the tenancy tab.

I am not sure if that is for docs under an `enable` section, or whether we just point them to pricing and they contact us and we tell them that information

Comment (CamrynCarter) [reply content/docs/overview/identity-and-access-management/index.md:77]: I think that the configuration options not showing are covered by the message in the UI, which directs people to RKVST support (for verified domain) and the pricing page (for ESSO). 

I did add a note that they must have a verified domain before setting up SSO, that was a good catch.

Comment (honourfish) [reply content/docs/overview/identity-and-access-management/index.md]: i agree :) great stuff!

**Review by kjc-rkvst — APPROVED** [review]

**Pull request merged**